### DISCUSSION TITLE
Can I run Keycloak embedded in Quarkos

### DISCUSSION THREAD

**Discussion by vasicvuk** [keycloak/keycloak#13941] [Q&A]
Is there a possibility to create Java project based on Quarkos and embed keycloak server within it, and also keep different extensions for Keycloak within same Java codebase.

**Comment by stianst** [comment]
In theory that is possible, but not something we'd officially support/recommend

**Reply by vasicvuk** [reply]
I think this would be an interesting feature as it would allow someone to easily extend KeyCloak, and integrate it with their application easier.
